functions_framework_builder/pubspec.yaml

```diff
@@ -13,7 +13,8 @@ dependencies:
   dart_style: ^1.3.10
   # There is a tight version constraint because the builder has a strict
   # dependency on all features exposed.
-  functions_framework: '>=0.3.0 <0.3.1'
+  # TODO: put back version tight version constraint before publish, ex: '>=0.3.0 <0.3.2'
```

Comment:
Change the version of this to `0.3.1-dev` ⬆️  now

And change the version constraint to `functions_framework: '>=0.3.0 <0.3.2'`